app/models/miq_widget.rb

```diff
@@ -60,8 +60,9 @@ def last_run_on
 
   def status
     if miq_task.nil?
-      return "None" if last_run_on.nil?
-      return "Complete"
+      return N_("None") if last_run_on.nil?
+
+      return N_("Complete")
     end
     miq_task.human_status
```

Comment:
You also need to address the case when associated `miq_task` isn't `nil`. Those values need to be marked for string collection as well.

Reply:
done